interface/modules/custom_modules/oe-module-claimrev-connect/src/ClaimSearchModel.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
header here and all other files below that are missing header

Reply:
done